src/occupational_classification/_config/main.py

```diff
@@ -79,7 +85,7 @@ def get_config(config_name: Optional[Union[Path, str]] = "config.toml") -> dict:
     Raises:
         FileNotFoundError: If the config file or required data not found.
     """
-    global _config
+    global _config  # noqa: PLW0603
```

Comment:
We need to be careful with the use of global especially for things that might have similar variable names (see [link](https://docs.astral.sh/ruff/rules/global-statement/)). Do you mind creating an issue for this in the repo just so that we don't forget?